cdap-app-fabric/src/test/java/co/cask/cdap/gateway/handlers/AuthorizationHandlerTest.java

```diff
@@ -95,22 +99,23 @@ public void testDisabled() throws IOException, UnauthorizedException {
         .build());
 
     NamespaceId ns1 = Ids.namespace("ns1");
+    Principal admin = new Principal("admin", Principal.PrincipalType.USER);
     try {
-      client.authorized(ns1, "admin", ImmutableSet.of(Action.READ));
+      client.authorized(ns1, admin, Action.READ);
       Assert.fail();
     } catch (IOException e) {
       Assert.assertTrue(e.getMessage().contains("404"));
```

Comment:
nit: Will be nice to add comment as why we expect 404 else the user has to read the whole test case to understand without any help